docs/operator-docs/naming-executions.md

```diff
@@ -26,13 +26,18 @@ In the following excerpt, the `QueueGranules` `config.executionNamePrefix`
 property is set using the value configured in the workflow's
 `meta.executionNamePrefix`.
 
+**Please note**: This `meta` property should not be confused with the rule's `executionNamePrefix` from the previous section.
```

Comment:
I'm not sure I understand this note. The `meta` property in the workflow will get its value for `executionNamePrefix` from `rule.meta`, if set

Reply:
We want to make it clear that setting the 'executionNamePrefix' directly at the root level of the rule, sets it for the WF executed by the rule, the other, in meta, sets it in the Message.
![rule1](https://user-images.githubusercontent.com/32745414/122784220-82a53500-d26f-11eb-85dd-2abb1678ccd4.png)

Reply:
@schwabm  is correct - the confusion/lack of clarity this is trying to address is the docs use a snippet from meta right after an example that is using the root key on the `rule` object.

Reply:
```suggestion
**Please note**: This `meta.executionNamePrefix` property should not be confused with the optional rule `executionNamePrefix` property from the previous section. Setting `executionNamePrefix` as a root property of the rule will set a prefix for the names of any workflows triggered by the rule. Setting `meta.executionNamePrefix` on the rule will set `meta.executionNamePrefix` in the workflow messages generated for this rule, allowing workflow steps like `QueueGranules` to read from the message `meta.executionNamePrefix` for their config. Then, workflows scheduled by `QueueGranules` would use the configured execution name prefix.
```

Thanks for clarifying @schwabm @Jkovarik . That is definitely confusing. I'd like to suggest this minor rewrite to call out the difference in the properties and their functionality a bit more directly

Reply:
Looks good. Thanks